### PR TITLE
Game window, menu UI and routing essentials

### PR DESCRIPTION
**Pull request by ugur-akin**
Implemented a skeleton for the UI including:
- A game window placed at `absolute (0,0)`, containing the whole screen (`100vw`,`100vh`),
- A layout including game title on top, an Icon menu at the bottom and the content in the middle,
- And Main Menu and Level Selection pages (final implementation of the level selector design is not included in this feature).

Implemented fundamentals of routing.

### CODE REVIEW

File: src/components/ui/IconMenu.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React, {useState} from 'react';
+import HomeIcon from '@material-ui/icons/HomeSharp';
+import SoundOffIcon from '@material-ui/icons/VolumeOffSharp';
+import SoundOnIcon from '@material-ui/icons/VolumeUpSharp';
+import HelpIcon from '@material-ui/icons/HelpOutlineSharp';
+import SettingsIcon from '@material-ui/icons/SettingsSharp';
```
Comment (ThinkTink) [inline src/components/ui/IconMenu.jsx]: import { HomeSharp as HomeIcon, VolumeOffSharp as SoundOffIcon } from '@material-ui/icons;

File: src/components/GameWindow.jsx [hunk 2]
```diff
@@ -1,29 +1,36 @@
...
+//    for different devices.
 
+const GameWindow = () => {
   return (
-    <CenteringBox>
-      <Box id="game-window">{windowContent}</Box>
+    <CenteringBox
+      id="game-window"
+      height="100vh"
+      width="100vw"
```
Comment (ugur-akin) [inline src/components/GameWindow.jsx]: Refer to [this stack-overflow answer](https://stackoverflow.com/a/56254633) for a potentially better way of doing this. Also consider hiding overflows with `overflow: hidden`.

File: src/components/ui/LevelButton.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {IconButton, makeStyles} from '@material-ui/core';
+
+const useStyles = makeStyles({
+  root: {
+    border: 'solid 3.3px #000000A0',
+    backgroundColor: '#00A0DF00',
```
Comment (ugur-akin) [inline src/components/ui/LevelButton.jsx]: Consider replacing these hexcodes with `rgba()` calls?

File: src/pages/MenuLayout.jsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+import IconMenu from '../components/ui/IconMenu';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    backgroundImage: `url(${bgImage})`,
+    backgroundPosition: 'center',
+    backgroundRepeat: 'no-repeat',
+    backgroundSize: 'cover',
+    boxSizing: 'border-box',
+    padding: theme.spacing(4),
```
Comment (ugur-akin) [inline src/pages/MenuLayout.jsx:14]: Should we move this to `div#screen`? Is there a screen display we don't want padding on?

File: src/themes/theme.js [hunk 5]
```diff
@@ -4,13 +4,13 @@ const baseThemeSettings = {
...
-      },
-    },
-  },
+  // overrides: {
+  //   MuiButton: {
+  //     root: {
+  //       textTransform: 'capitalize',
+  //     },
+  //   },
+  // },
```
Comment (ugur-akin) [inline src/themes/theme.js]: We should remove these.

**Review by ugur-akin — COMMENTED** [review]
Consider moving prop-styling used on `Box` components to classes. Maybe it makes sense to make an exception for `flex` related styles as a convention.

**Pull request merged**